docs/sources/explore/trace-integration.md

```diff
@@ -8,167 +7,159 @@ labels:
     - cloud
     - enterprise
     - oss
-title: Tracing in Explore
+title: Traces in Explore
 weight: 20
 ---
 
-# Tracing in Explore
+# Traces in Explore
 
-You can use Explore to query and visualize traces from tracing data sources.
+You can use Explore to query and visualize traces from tracing data sources. Supported data sources include:
 
-Supported data sources are:
-
-- [Tempo]({{< relref "../datasources/tempo/" >}}) (supported ingestion formats: OpenTelemetry, Jaeger, and Zipkin)
-- [Jaeger]({{< relref "../datasources/jaeger/" >}})
-- [Zipkin]({{< relref "../datasources/zipkin/" >}})
+- [Tempo](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/)
+- [Jaeger](/docs/grafana/<GRAFANA_VERSION>/datasources/jaeger/)
+- [Zipkin](/docs/grafana/<GRAFANA_VERSION>/datasources/zipkin/)
 - [X-Ray](https://grafana.com/grafana/plugins/grafana-x-ray-datasource)
-- [Azure Monitor Application Insights]({{< relref "../datasources/azure-monitor/" >}})
+- [Azure Monitor](/docs/grafana/latest/datasources/azure-monitor/)
 - [ClickHouse](https://github.com/grafana/clickhouse-datasource)
-- [New Relic](https://grafana.com/grafana/plugins/grafana-newrelic-datasource)
-- [Infinity](https://grafana.com/grafana/plugins/yesoreyeram-infinity-datasource)
+- [New Relic](/docs/plugins/grafana-newrelic-datasource/latest/)
+- [Infinity](/docs/plugins/yesoreyeram-infinity-datasource/latest/)
 
-For information on how to configure queries for the data sources listed above, refer to the documentation for specific data source.
+If you are getting started working with traces in Tempo refer to the following documents:
 
-## Query editor
+- [Get started with Grafana Tempo](docs/<TEMPO_VERSION>/latest/getting-started/)
+- [Trace structure](https://grafana.com/docs/tempo/<TEMPO_VERSION>/traceql/trace-structure/#trace-structure)
+- [How TraceQL works](/docs/tempo/<TEMPO_VERSION>/traceql/architecture/)
+- [Best practices for tracing](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/tracing-best-practices/)
```

Comment:
Because this is Grafana doc, what do you think about shifting the focus from Tempo docs to the general information about tracing? Also, the TraceQL content is specific to only Tempo. Here are some suggestions instead of the links you have below. 

```suggestion
- [Introduction to tracing](https://grafana.com/docs/tempo/<TEMPO_VERSION>/introduction/)
- [Trace structure](https://grafana.com/docs/tempo/<TEMPO_VERSION>/traceql/trace-structure/#trace-structure)
- [Traces and telemetry](https://grafana.com/docs/tempo/<TEMPO_VERSION>/introduction/telemetry/)
- [Using traces to find solutions to problems](https://grafana.com/docs/tempo/<TEMPO_VERSION>/introduction/solutions-with-traces/)
- [Best practices for tracing](/docs/grafana/<GRAFANA_VERSION>/datasources/tempo/tracing-best-practices/)
```